### ISSUE TITLE
where is label_2  in test dataset of KITTI?

### ISSUE THREAD

**Issue by 94happy** [jeasinema/VoxelNet-tensorflow#44]
Can you provide it  for me?Thanks!

**Comment by happiensslz** [comment]
ok,I know!Thanks!

**Comment by wutaoo** [comment]
Is the label_2 folder under testing an empty folder？

**Comment by yifanwz** [comment]
Where is label_2 in test dataset?

**Comment by abhigoku10** [comment]
@happiensslz  where is the testing label_2 folder???

**Comment by liyang0522** [comment]
due to the kitti privacy,the testing label is not provided!    Follow the paper do, split the training into train and test!

**Comment by mtli77** [comment]
@happiensslz  where is the testing label_2 folder?
